lms/static/scripts/frontend_apps/components/LMSFilePicker.js

```diff
@@ -1,10 +1,11 @@
 import { LabeledButton } from '@hypothesis/frontend-shared';
-import { createElement } from 'preact';
+import { createElement, Fragment } from 'preact';
```

Comment:
```suggestion
import { createElement } from 'preact';
```

You can use `<></>`

Reply:
I think you still need to import `Fragment` in order to use `<>` in this project as it hasn't been switched over to use the new JSX processing method.

Reply:
Correct. I started by trying the `<>` variant and found it was not supported here.